vhosting/vhosting_check.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+import argparse
+import urllib.request as req
+import os, re, requests, sys
+
+banners_dir = "./banners/"
```

Comment:
let's figure out a way to avoid this, i don't think we should check the banners into utils